tests/test_file_storage.py

```diff
@@ -0,0 +1,650 @@
+import io
+import json
+import os.path
+from json import JSONDecodeError
+
+import pdfplumber
+import pytest
+from dotenv import load_dotenv
+
+from unstract.sdk.constants import MimeType
+from unstract.sdk.exceptions import FileOperationError
+from unstract.sdk.file_storage import FileStorage, FileStorageProvider
+
+load_dotenv()
+
+
+class TEST_CONSTANTS:
+    READ_FOLDER_PATH = os.environ.get("READ_FOLDER_PATH")
+    WRITE_FOLDER_PATH = os.environ.get("WRITE_FOLDER_PATH")
+    RECURSION_FOLDER_PATH = os.environ.get("RECURSION_FOLDER_PATH")
+    READ_PDF_FILE = os.environ.get("READ_PDF_FILE")
+    READ_TEXT_FILE = os.environ.get("READ_TEXT_FILE")
+    WRITE_PDF_FILE = os.environ.get("WRITE_PDF_FILE")
+    WRITE_TEXT_FILE = os.environ.get("WRITE_TEXT_FILE")
+    TEST_FOLDER = os.environ.get("TEST_FOLDER")
+    GCS_BUCKET = os.environ.get("GCS_BUCKET")
+    TEXT_CONTENT = os.environ.get("TEXT_CONTENT")
+    FILE_STORAGE_GCS = "FILE_STORAGE_GCS"
+    FILE_STORAGE_MINIO = "FILE_STORAGE_MINIO"
+    FILE_STORAGE_LOCAL = "FILE_STORAGE_LOCAL"
+
+
+def file_storage(provider: FileStorageProvider):
+    try:
+        if provider == FileStorageProvider.GCS:
+            creds = json.loads(os.environ.get(TEST_CONSTANTS.FILE_STORAGE_GCS, {}))
+        elif provider == FileStorageProvider.Minio:
+            creds = json.loads(os.environ.get(TEST_CONSTANTS.FILE_STORAGE_MINIO, {}))
+        elif provider == FileStorageProvider.Local:
+            creds = json.loads(os.environ.get(TEST_CONSTANTS.FILE_STORAGE_LOCAL, {}))
```

Comment:
creds = json.loads(os.environ.get(TEST_CONSTANTS.FILE_STORAGE_LOCAL, ""))

Reply:
Is this the same  comment as above @kirtimanmishrazipstack ?

Reply:
creds = json.loads(os.environ.get(TEST_CONSTANTS.FILE_STORAGE_LOCAL, ""))

Reply:
@gaya3-zipstack can you do like this, else the test cases break

Reply:
yes , can you make it like the comment above?